docs/book/releases/v2.3.0.md

```diff
@@ -0,0 +1,69 @@
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver - v2.2.1 release
+
+## New Feature
+
+- There is no new feature released in v2.2.1.  v2.2.1 is the patch release to fix critical issue observed in [v2.2.0](v2.2.0.md) release.
```

Comment:
Looks like this page is accidentally added here.

Reply:
Yes, I will create a separate PR once release is complete to update known issues. Removing this for now

Reply:
Can you ask @RaunakShah to add known issues in this PR - https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/1184

Reply:
Makes sense: https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/1184/files#r675901952
cc: @RaunakShah ^^